File: openapi-generator/scala-gatling/src/gatling/scala/org/openapitools/client/model/InlineResponse2008.scala

package org.openapitools.client.model


case class InlineResponse2008 (
    _sensors: Option[List[Sensor]]
)
object InlineResponse2008 {
    def toStringBody(var_sensors: Object) =
        s"""
        | {
        | "sensors":$var_sensors
        | }
        """.stripMargin
}

File: openapi-generator/kotlin-spring/src/main/kotlin/org/openapitools/model/InlineResponse2008.kt
package org.openapitools.model

import java.util.Objects
import com.fasterxml.jackson.annotation.JsonProperty
import org.openapitools.model.Sensor
import javax.validation.Valid
import javax.validation.constraints.*

/**
 * 
 * @param sensors 
 */
data class InlineResponse2008 (

        @JsonProperty("sensors") val sensors: List<Sensor>? = null
) {

}

